File: video-uploader/src/main/java/org/breizhcamp/video/uploader/dto/Event.java
package org.breizhcamp.video.uploader.dto;

import com.fasterxml.jackson.annotation.JsonProperty;

import java.time.LocalDateTime;

/**
 * JSON deserialization of an Event
 */
public class Event {

	private String id;
	private String name;
	private String description;
	private String speakers;
	private String language;
	@JsonProperty("event_start")
	private LocalDateTime eventStart;

	@JsonProperty("event_type")
	private String eventType;
	private String format;
	private String venue;

	public String getId() {
		return id;
	}

	public void setId(String id) {
		this.id = id;
	}

	public String getName() {
		return name;
	}

	public void setName(String name) {
		this.name = name;
	}

	public String getDescription() {
		return description;
	}

	public void setDescription(String description) {
		this.description = description;
	}

	public String getSpeakers() {
		return speakers;
	}

	public void setSpeakers(String speakers) {
		this.speakers = speakers;
	}

	public String getLanguage() {
		return language;
	}

	public void setLanguage(String language) {
		this.language = language;
	}

	public LocalDateTime getEventStart() {
		return eventStart;
	}

	public void setEventStart(LocalDateTime eventStart) {
		this.eventStart = eventStart;
	}

	public String getEventType() {
		return eventType;
	}

	public void setEventType(String eventType) {
		this.eventType = eventType;
	}

	public String getFormat() {
		return format;
	}

	public void setFormat(String format) {
		this.format = format;
	}

	public String getVenue() {
		return venue;
	}

	public void setVenue(String venue) {
		this.venue = venue;
	}
}

File: src/main/java/org/breizhcamp/video/uploader/dto/Event.kt
package org.breizhcamp.video.uploader.dto

import com.fasterxml.jackson.annotation.JsonProperty

import java.time.LocalDateTime

/**
 * JSON deserialization of an Event
 */
class Event {
    var id: Integer? = null
    var name: String? = null
    var description: String? = null
    var speakers: String? = null
    var language: String? = null
    @JsonProperty("event_start")
    var eventStart: LocalDateTime? = null
    @JsonProperty("event_end")
    var eventEnd: LocalDateTime? = null
    @JsonProperty("event_type")
    var eventType: String? = null
    var format: String? = null
    var venue: String? = null
    @JsonProperty("venue_id")
    var venueId: String? = null
    @JsonProperty("video_url")
    var videoUrl:String? = null
    @JsonProperty("files_url")
    var filesUrl:String? = null
    @JsonProperty("slides_url")
    var slidesUrl: String? = null
}
